src/main/java/seedu/address/model/client/Client.java

```diff
@@ -87,8 +85,8 @@ public LastModifiedInstant getLastModifiedInstant() {
      *
      * @return The list of client notes associated with this client.
      */
-    public Set<Note> getClientNotes() {
-        return Collections.unmodifiableSet(this.clientNotes);
+    public List<Note> getClientNotes() {
+        return Collections.unmodifiableList(this.clientNotes);
     }
```

Comment:
This could be a duplicated method of the getClientNotesAsList

Reply:
Would suggest changing this method name to `getClientNotesAsUnmodifiableList()`

Reply:
changed to `getClientNotesAsUnmodifiableList()`

Reply:
also deprecated the `getClientNotesAsList`